service/src/main/java/greencity/service/ubs/UBSClientServiceImpl.java

```diff
@@ -1952,10 +1953,14 @@ public Set<String> getAllAuthorities(String email) {
     }
 
     @Override
-    public void updateEmployeesAuthorities(UserEmployeeAuthorityDto dto, String email) {
+    public void updateEmployeesAuthorities(UserEmployeeAuthorityDto dto) {
         Employee employee = employeeRepository.findByEmail(dto.getEmployeeEmail())
             .orElseThrow(() -> new NotFoundException(EMPLOYEE_DOESNT_EXIST));
-        userRemoteClient.updateEmployeesAuthorities(dto, email);
+        try {
+            userRemoteClient.updateEmployeesAuthorities(dto);
+        } catch (HystrixRuntimeException e) {
+            throw new BadRequestException(EMPLOYEE_DOESNT_EXIST);
```

Comment:
I think we should check the response core and throw `BadRequestException(EMPLOYEE_DOESNT_EXIST)` only if the response code is `404`

Reply:
done